corebehrt/functional/load.py

```diff
@@ -0,0 +1,40 @@
+""" Utils for loading data """
+
+from typing import List, Union
+import torch
+import os
+from os.path import join
+
+
+# Taken from common.loader
+def load_pids(files: Union[List, str]) -> List:
+    """Loads pids from multiple files or one file. Doesn't preserve order."""
+    if isinstance(files, str):
+        return set(torch.load(files))
+    pids = set()
+    for file in files:
+        pids.update(set(torch.load(file)))
+    return pids
+
+
+def load_predefined_pids(
```

Comment:
Suggest using `pathlib.Path` instead to easier check for suffixes and `is_dir` - but this is general for the entire codebase

Reply:
Same here